lib/slice.ark

```diff
@@ -0,0 +1,16 @@
+{
+  (let slice (fun (string escapeCharacter){
```

Comment:
Reviewing for coding style: put a space after the argument list: `(fun (arg1 arg2...) {`

Reply:
Also, should probably rename the function `split`, since `slice` is usually used for arrays, not string. Since the name of the function should match the file, better renaming it `Split.ark` (mind the uppercase letter)

Reply:
The `escapeCharacter` isn't right, this isn't a `\n` or a `\b`, it's a separator

Reply:
rename it to separator

Reply:
renamed the file to Slice and put the space